src/radical/pilot/tmgr/staging_input/default.py

```diff
@@ -68,7 +68,8 @@ def initialize(self):
         # we subscribe to the command channel to learn about pilots being added
         # to this task manager.
         self.register_subscriber(rpc.CONTROL_PUBSUB, self._base_command_cb)
-
+        self.TASK_BULK_MKDIR_THRESHOLD = self.cfg.get('task_bulk_mkdir_threshold',
+                     TASK_BULK_MKDIR_THRESHOLD)
```

Comment:
class member variables should be lower case and short (only constants are all uppercase).  Also, private variables should have a leading underscrore. Maybe use `self._mkdir_threshold`?